### ISSUE TITLE
Monorepo packages that can be import by supabase functions

### ISSUE THREAD

**Issue by selbyk** [psteinroe/supasample#41]
I'm currently trying to convert our many repos into one monorepo, and we would like to have monorepo packages that can be imported by our supabase functions.

Have you had any luck doing this or see a path towards this?

**Comment by psteinroe** [comment]
hey! thanks for reaching out. Unfortunately, I have neither used supabase edge functions nor deno, so I won't be of much help here. But happy to add this here if we get this solved.

**Comment by tonven** [comment]
@selbyk did you manage to integrate edge function into monorepo?

**Comment by selbyk** [comment]
@tonven no, we ended up migrating away from supabase altogether and instead run pg on RDS and a traditional api service (and a few others) on ECS.

deno not supporting aync_hooks made it impossible to do a few things we needed.
